qemu/qemu.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// Ctx is not used at the moment.
 	Ctx context.Context
 
+	// Name is the qemu guest name
+	Name string
+	
```

Comment:
Surprised go fmt didn't autofix this, but you've got trailing whitespace here on this line.